Doc/library/symtable.rst

```diff
@@ -123,7 +123,7 @@ Examining Symbol Tables
 
 .. class:: Class
 
-   A namespace of a class.  This class inherits :class:`SymbolTable`.
+   A namespace of a class.  This class inherits from :class:`SymbolTable`.
```

Comment:
Another one in this file on line 100.

Reply:
Got it. Not sure how I missed it. Maybe I was looking for a space after...